### PR TITLE
Remove Jupyter notebooks from documentation

### PR DESCRIPTION
**Pull request by paulromano**
We now have our Jupyter notebooks maintained in a [separate repository](https://github.com/openmc-dev/openmc-notebooks) along with a [wiki page](https://github.com/openmc-dev/openmc/wiki/Example-Jupyter-Notebooks) that links to those notebooks as well as other user-contributed notebooks. This PR removes the example notebooks from our main docs on docs.openmc.org and replaces them with a link to the wiki page, which will be easier to maintain going forward.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,9 +34,6 @@ build
 # build from src/utils/setup.py
 src/utils/build
 
-# xml-fortran reader
```
Comment (nelsonag) [inline .gitignore:38]: The last of an era..

**Review by nelsonag — CHANGES_REQUESTED** [review]
Good change @paulromano 
One request from building this:
- source/pythonapi/index.rst still has a reference to examples in the first paragraph :ref:"examples") (single quotes changed to double to avoid code blocks). Might as well make that a hyperlink to the new wiki location.

**Comment by paulromano** [comment]
Good call @nelsonag. Just made that update

**Review by nelsonag — APPROVED** [review]
Looks good to me, thanks @paulromano!

**Pull request merged**